src/network/networkd-ndisc.c

```diff
@@ -1003,7 +1015,7 @@ int config_parse_address_generation_type(
         assert(data);
 
         if (isempty(rvalue)) {
-                network->ipv6_tokens = ordered_hashmap_free(network->ipv6_tokens);
+                network->ipv6_tokens = ordered_set_free_free(network->ipv6_tokens);
```

Comment:
I'd prefer to have the hash ops defined with a free function, and then just use `ordered_set_free` here and in other places where the set is freed.

Reply:
OK.